src/qml/models/nodemodel.cpp

```diff
@@ -66,25 +88,118 @@ void NodeModel::setRemainingSyncTime(double new_progress)
         }
         if (remainingMSecs > 0 && m_block_process_time.count() % 1000 == 0) {
             m_remaining_sync_time = remainingMSecs;
+            setFormattedRemainingSyncTime(m_remaining_sync_time);
 
             Q_EMIT remainingSyncTimeChanged();
         }
         static const int MAX_SAMPLES = 5000;
         if (m_block_process_time.count() > MAX_SAMPLES) {
             m_block_process_time.remove(1, m_block_process_time.count() - 1);
         }
+    } else {
+        m_remaining_sync_time = 0;
+        setFormattedRemainingSyncTime(m_remaining_sync_time);
+        
+        Q_EMIT remainingSyncTimeChanged();
```

Comment:
Please remove trailing spaces to fix the CI lint job:
```suggestion

```
```suggestion
        setFormattedRemainingSyncTime(m_remaining_sync_time);

        Q_EMIT remainingSyncTimeChanged();
```